aspnet/fundamentals/request-features.rst

```diff
@@ -1,8 +1,177 @@
-.. include:: /../common/stub-topic.txt
+Request Features
+================
 
-|stub-icon| Request Features
-============================
+By `Steve Smith`_
 
-.. include:: /../common/stub-notice.txt
+Individual web server features related to how HTTP requests and responses are handled have been factored into separate interfaces, defined in the `HttpAbstractions <https://github.com/aspnet/HttpAbstractions>`_ package. These abstractions are used by individual server implementations and middleware to create and modify the application's hosting pipeline.
```

Comment:
HttpAbstractions is a github repository, not a nuget package. The package is Microsoft.AspNet.Http.Features.